torchvision/transforms/transforms.py

```diff
@@ -307,13 +307,11 @@ class Resize(torch.nn.Module):
             ``InterpolationMode.BILINEAR`` and ``InterpolationMode.BICUBIC`` are supported.
             The corresponding Pillow integer constants, e.g. ``PIL.Image.BILINEAR`` are accepted as well.
         max_size (int, optional): The maximum allowed for the longer edge of
-            the resized image: if the longer edge of the image is greater
-            than ``max_size`` after being resized according to ``size``, then
-            the image is resized again so that the longer edge is equal to
-            ``max_size``. As a result, ``size`` might be overruled, i.e. the
-            smaller edge may be shorter than ``size``. This is only supported
-            if ``size`` is an int (or a sequence of length 1 in torchscript
-            mode).
+            the resized image. If the longer edge of the image is greater
+            than ``max_size`` after being resized according to ``size``, 
+            ``size`` will be overruled. As a result, the smaller edge may be 
```

Comment:
```suggestion
            ``size`` will be overruled so that the longer edge is equal to
            ``max_size``.
            As a result, the smaller edge may be 
```